tests/MongoDB.Driver.Core.Tests/Core/Configuration/ConnectionSettingsTests.cs

```diff
@@ -138,7 +150,9 @@ public void constructor_with_maxIdleTime_should_initialize_instance()
             var subject = new ConnectionSettings(maxIdleTime: maxIdleTime);
 
             subject.ApplicationName.Should().BeNull();
-            subject.Authenticators.Should().Equal(__defaults.Authenticators);
+            subject.AuthenticatorFactories.Should().NotBeNull();
+            var authenticators = subject.AuthenticatorFactories.Select(a => a.Create());
+            authenticators.Should().BeEmpty();
```

Comment:
Replace 153-155 with:

            subject.AuthenticatorFactories.Should().BeEmpty();